fiftyone/factory/repos/delegated_operation.py

```diff
@@ -355,8 +355,6 @@ def list_operations(
         if paging.limit:
             docs = docs.limit(paging.limit)
 
-        return [DelegatedOperationDocument().from_pymongo(doc) for doc in docs]
```

Comment:
This looks like a failed merge resolution

Reply:
No idea what you're talking about 😇